compiler/src/Language/Bond/Codegen/Cpp/ApplyOverloads.hs

```diff
@@ -22,20 +23,22 @@ data Protocol =
 
 
 -- Apply overloads
-applyOverloads :: [Protocol] -> Text -> Text -> Declaration -> Text
-applyOverloads protocols attr body Struct {..} | null declParams = [lt|
+applyOverloads :: [Protocol] -> MappingContext -> Text -> Text -> Declaration -> Text
+applyOverloads protocols cpp attr body s@Struct {..} | null declParams = [lt|
     //
     // Overloads of Apply function with common transforms for #{declName}.
```

Comment:
I originally did. I changed it back because I don't think it's ambiguous to a human, and if it is, the qualified names in the code make it clear which Foo we're talking about.